adapter-base/pom.xml

```diff
@@ -83,6 +83,10 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-config-yaml</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-logging-gelf</artifactId>
+    </dependency>
```

Comment:
If this is a component that is only needed in the container images, then this should go into `adapters/base-quarkus` instead FMPOV